quetz/main.py

```diff
@@ -912,7 +912,8 @@ def post_package(
     auth: authorization.Rules = Depends(get_rules),
     dao: Dao = Depends(get_dao),
 ):
-    user_id = auth.assert_user()
+    owner_id = auth.assert_owner()
```

Comment:
Since the argument to `create_package` still requires us to pass in a `user_id`, I'd recommend to still keep the line as `user_id = auth.assert_owner()` here, and add a comment that explains why we are using the user id of the owner here.